doc/reference/bluetooth/hci.txt

```diff
@@ -1014,6 +1015,37 @@ by the controller for the requested Handle_Type and Handle.
 When the Read_Tx_Power_Level command has completed, a Command Complete
 event shall be generated.
 
+Zephyr Read USB Transport Mode Command
+======================================
+
+This command read the supported USB transport modes.
+
++--------------------------+-------+--------------------+--------------------+
+| Command                  | OCF   | Command            | Return             |
+|                          |       | Parameters         | Parameters         |
++--------------------------+-------+--------------------+--------------------+
+| Read_USB_Transport_Mode  | 0x010 |                    | Status             |
+|                          |       |                    | Num_Supported_Modes|
+|                          |       |                    | Supported_Mode[i]  |
++--------------------------+-------+--------------------+--------------------+
+
+When the Read_USB_Transport_Mode command has completed, a Command Complete
+event shall be generated.
+
+Zephyr Set USB Transport Mode Command
+=====================================
+
+This command set the host transport mode and reset the controller.
```

Comment:
Sure, I thought that would be obvious but I can spell out what reset it would be doing.

Reply:
Thanks. 

Is the transport mode retained over regular HCI Reset command? It would be good to clarify the behavior for that as well. Not sure which option is best here.

Reply:
HCI reset resets just the HCI portion, H:2/H:4 is bellow that so it shouldn't be affected by it, we do reset in order to start fresh and don't have any pending events, etc, sending an HCI Reset won't affect the mode.